{"name": "test_broken", "status": "broken", "statusDetails": {"message": "Exception: oops", "trace": "def test_broken():\n>       raise Exception('oops')\nE       Exception: oops\n\nallure_demo1.py:19: Exception"}, "start": 1655274598063, "stop": 1655274598063, "uuid": "d3486b5a-95f0-4713-82b9-f1be799483c0", "historyId": "910a4c6686f2243edeff3d85b787b1ba", "testCaseId": "2418e8d31318733aaa3199625cc8b6c1", "fullName": "allure_demo1#test_broken", "labels": [{"name": "suite", "value": "allure_demo1"}, {"name": "host", "value": "MacBook-Pro-6.local"}, {"name": "thread", "value": "19317-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "allure_demo1"}]}